java-cfenv-tests-boot/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id 'io.pivotal.cfenv.java-conventions'
+}
+
+description = 'Java CF Env Spring Boot 3 specific tests'
+
+java {
```

Comment:
I think this is not necessary, should be applied by `id 'io.pivotal.cfenv.java-conventions'` 🤔 At least none of the other modules define it.

Reply:
Now I see that also applies to the dependencies sections, some could be removed since they are already applied.

Reply:
ok I removed it - used to be useful when both 2.x and 3.x support were together